CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 3.1.0
   - Feat: event `target => namespace` support (for ECS) [#37](https://github.com/logstash-plugins/logstash-codec-json/pull/37)
   - Refactor: dropped support for old Logstash versions (< 6.0)
+  - Fix: json parsing compatibility + freeze event.original value [#38](https://github.com/logstash-plugins/logstash-codec-json/pull/38)
```

Comment:
I only would be more specific on the fix description, citing that it's for backward compatibility, where "" is parsed as []

Reply:
```suggestion
  - Fix: json parsing compatibility (when parsing blank strings) + freeze event.original value [#38](https://github.com/logstash-plugins/logstash-codec-json/pull/38)
```

Reply:
Thanks @kares :-) :+1: